### PR TITLE
JUnit5 :: server & transform packages

### PR DESCRIPTION
**Pull request by dev-mlb**

### CODE REVIEW

**Review by ukengineer925 — APPROVED** [review]

File: src/test/java/emissary/output/roller/JournaledCoalescerTest.java [hunk 1]
```diff
@@ -204,13 +202,13 @@ public void testAddFilesWithRoll() throws Exception {
         }
         // verify
         Path bud1Destination = targetBUDPath.resolve(expectedPrefix1);
-        assertThat("Expected target to exist " + bud1Destination.toString(), Files.exists(bud1Destination), equalTo(true));
-        assertThat(bud1Destination.getFileName().toString(), not(equalTo(expectedPrefix2)));
-        assertThat(bud1Destination.getParent(), equalTo(targetBUDPath));
-        assertThat(bud1Destination.getFileName().toString(), equalTo(expectedPrefix1));
-        assertThat(Files.readAllLines(bud1Destination, Charset.defaultCharset()), equalTo(BUD1_LINES));
+        assertTrue(Files.exists(bud1Destination), "Expected target to exist " + bud1Destination);
+        assertNotEquals(bud1Destination.getFileName().toString(), expectedPrefix2);
```
Comment (drivenflywheel) [inline src/test/java/emissary/output/roller/JournaledCoalescerTest.java:206]: Swap positions of unexpected and actual
```suggestion
        assertNotEquals(expectedPrefix2, bud1Destination.getFileName().toString());
```

File: src/test/java/emissary/server/EmissaryServerIT.java [hunk 2]
```diff
@@ -42,7 +41,7 @@ public void testSSLWorks() throws Exception {
             String endpoint = cmd.getScheme() + "://" + hostPort + "/api/version";
             MapResponseEntity versionMap = client.send(new HttpGet(endpoint)).getContent(MapResponseEntity.class);
             Map<String, String> response = versionMap.getResponse();
-            assertThat(response.get(hostPort), equalTo(new Version().getVersion()));
+            assertEquals(new Version().getVersion(), response.get(hostPort));
         } catch (Exception e) {
```
Comment (drivenflywheel) [inline src/test/java/emissary/server/EmissaryServerIT.java:45]: Optional cleanup:   discard the `catch` clause, which could be masking test failures

File: src/test/java/emissary/server/InitializeContextTest.java [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
-import org.junit.Ignore;
-import org.junit.Test;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
 
-public class InitializeContextTest extends UnitTest {
+class InitializeContextTest extends UnitTest {
 
-    @Ignore
+    @Disabled("Not yet implemented")
```
Comment (drivenflywheel) [inline src/test/java/emissary/server/InitializeContextTest.java:9]: Should either delete this placeholder file or create a follow-on ticket to implement tests

File: src/test/java/emissary/server/api/EnvTest.java [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
-import org.junit.Ignore;
-import org.junit.Test;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
 
-public class EnvTest extends EndpointTestBase {
+class EnvTest extends EndpointTestBase {
 
-    @Ignore
+    @Disabled("Not yet implemented")
```
Comment (drivenflywheel) [inline src/test/java/emissary/server/api/EnvTest.java:11]: Should either delete this placeholder file or create a follow-on ticket to implement tests

File: src/test/java/emissary/server/api/HealthCheckActionTest.java [hunk 5]
```diff
@@ -1,16 +1,16 @@
...
+import static org.junit.jupiter.api.Assertions.fail;
 
 import emissary.server.mvc.EndpointTestBase;
-import org.junit.Ignore;
-import org.junit.Test;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
 
-public class HealthCheckActionTest extends EndpointTestBase {
+class HealthCheckActionTest extends EndpointTestBase {
```
Comment (drivenflywheel) [inline src/test/java/emissary/server/api/HealthCheckActionTest.java:9]: Should either delete this placeholder file or create a follow-on ticket to implement tests

File: src/test/java/emissary/server/api/MetricsActionTest.java [hunk 6]
```diff
@@ -1,16 +1,16 @@
...
+import static org.junit.jupiter.api.Assertions.fail;
 
 import emissary.server.mvc.EndpointTestBase;
-import org.junit.Ignore;
-import org.junit.Test;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
 
-public class MetricsActionTest extends EndpointTestBase {
+class MetricsActionTest extends EndpointTestBase {
```
Comment (drivenflywheel) [inline src/test/java/emissary/server/api/MetricsActionTest.java:9]: Should either delete this placeholder file or create a follow-on ticket to implement tests

File: src/test/java/emissary/server/api/PoolTest.java [hunk 7]
```diff
@@ -1,16 +1,16 @@
...
+import static org.junit.jupiter.api.Assertions.fail;
 
 import emissary.server.mvc.EndpointTestBase;
-import org.junit.Ignore;
-import org.junit.Test;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
 
-public class PoolTest extends EndpointTestBase {
+class PoolTest extends EndpointTestBase {
```
Comment (drivenflywheel) [inline src/test/java/emissary/server/api/PoolTest.java:9]: Should either delete this placeholder file or create a follow-on ticket to implement tests

File: src/test/java/emissary/server/api/VersionTest.java [hunk 8]
```diff
@@ -1,16 +1,16 @@
...
+import static org.junit.jupiter.api.Assertions.fail;
 
 import emissary.server.mvc.EndpointTestBase;
-import org.junit.Ignore;
-import org.junit.Test;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
 
-public class VersionTest extends EndpointTestBase {
+class VersionTest extends EndpointTestBase {
```
Comment (drivenflywheel) [inline src/test/java/emissary/server/api/VersionTest.java:9]: Should either delete this placeholder file or create a follow-on ticket to implement tests

File: src/test/java/emissary/server/mvc/DumpDirectoryActionTest.java [hunk 9]
```diff
@@ -1,13 +1,13 @@
 package emissary.server.mvc;
 
-import org.junit.Ignore;
-import org.junit.Test;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
 
-public class DumpDirectoryActionTest extends EndpointTestBase {
+class DumpDirectoryActionTest extends EndpointTestBase {
```
Comment (drivenflywheel) [inline src/test/java/emissary/server/mvc/DumpDirectoryActionTest.java:6]: Should either delete this placeholder file or create a follow-on ticket to implement tests

File: src/test/java/emissary/server/mvc/EnvironmentActionTest.java [hunk 10]
```diff
@@ -1,13 +1,13 @@
 package emissary.server.mvc;
 
-import org.junit.Ignore;
-import org.junit.Test;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
 
-public class EnvironmentActionTest extends EndpointTestBase {
+class EnvironmentActionTest extends EndpointTestBase {
```
Comment (drivenflywheel) [inline src/test/java/emissary/server/mvc/EnvironmentActionTest.java:6]: Should either delete this placeholder file or create a follow-on ticket to implement tests

File: src/test/java/emissary/server/mvc/NamespaceActionTest.java [hunk 11]
```diff
@@ -1,13 +1,13 @@
 package emissary.server.mvc;
 
-import org.junit.Ignore;
-import org.junit.Test;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
 
-public class NamespaceActionTest extends EndpointTestBase {
+class NamespaceActionTest extends EndpointTestBase {
```
Comment (drivenflywheel) [inline src/test/java/emissary/server/mvc/NamespaceActionTest.java:6]: Should either delete this placeholder file or create a follow-on ticket to implement tests

File: src/test/java/emissary/server/mvc/ShutdownActionTest.java [hunk 12]
```diff
@@ -1,13 +1,13 @@
 package emissary.server.mvc;
 
-import org.junit.Ignore;
-import org.junit.Test;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
 
-public class ShutdownActionTest extends EndpointTestBase {
+class ShutdownActionTest extends EndpointTestBase {
```
Comment (drivenflywheel) [inline src/test/java/emissary/server/mvc/ShutdownActionTest.java:6]: Should either delete this placeholder file or create a follow-on ticket to implement tests

File: src/test/java/emissary/server/mvc/ThreadDumpActionTest.java [hunk 13]
```diff
@@ -1,13 +1,13 @@
 package emissary.server.mvc;
 
-import org.junit.Ignore;
-import org.junit.Test;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
 
-public class ThreadDumpActionTest extends EndpointTestBase {
+class ThreadDumpActionTest extends EndpointTestBase {
```
Comment (drivenflywheel) [inline src/test/java/emissary/server/mvc/ThreadDumpActionTest.java:6]: Should either delete this placeholder file or create a follow-on ticket to implement tests

File: src/test/java/emissary/server/mvc/TransferDirectoryActionTest.java [hunk 14]
```diff
@@ -1,13 +1,13 @@
 package emissary.server.mvc;
 
-import org.junit.Ignore;
-import org.junit.Test;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
 
-public class TransferDirectoryActionTest extends EndpointTestBase {
+class TransferDirectoryActionTest extends EndpointTestBase {
```
Comment (drivenflywheel) [inline src/test/java/emissary/server/mvc/TransferDirectoryActionTest.java:6]: Should either delete this placeholder file or create a follow-on ticket to implement tests

File: src/test/java/emissary/test/core/TestExtractionTest.java [hunk 15]
```diff
@@ -8,17 +10,17 @@
 import org.jdom2.Element;
 import org.jdom2.JDOMException;
 import org.jdom2.input.SAXBuilder;
-import org.junit.Assert;
-import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
```
Comment (drivenflywheel) [inline src/test/java/emissary/test/core/TestExtractionTest.java:13]: Replace with a static import of `org.junit.jupiter.api.Assertions.assertNotNull`

File: src/test/java/emissary/util/shell/ExecutrixTest.java [hunk 16]
```diff
@@ -228,7 +226,7 @@ public void testCleanupNonExistentDir() throws Exception {
         when(dirToDelete.exists()).thenReturn(false);
 
         final boolean result = Executrix.cleanupDirectory(dirToDelete);
-        assertThat(result, is(true));
+        assertTrue(result);
```
Comment (drivenflywheel) [inline src/test/java/emissary/util/shell/ExecutrixTest.java:229]: Consider reverting these changes in favor of the more sweeping changes to this file in PR #230

**Review by drivenflywheel — CHANGES_REQUESTED** [review]
Mostly minor changes suggested.  Biggest suggestion is removal of placeholder test classes that contain no implemented tests or creation of a follow-on issue to have those placeholder classes fleshed out.

File: src/test/java/emissary/output/roller/JournaledCoalescerTest.java (hunk 1 above)
Comment (dev-mlb) [reply src/test/java/emissary/output/roller/JournaledCoalescerTest.java:206]: I had to base all of the junit 5 upgrades on #223 -- I'll fix it in #236

File: src/test/java/emissary/server/InitializeContextTest.java (hunk 3 above)
Comment (dev-mlb) [reply src/test/java/emissary/server/InitializeContextTest.java:9]: Ticket created

File: src/test/java/emissary/server/api/EnvTest.java (hunk 4 above)
Comment (dev-mlb) [reply src/test/java/emissary/server/api/EnvTest.java:11]: Ticket created

File: src/test/java/emissary/server/api/HealthCheckActionTest.java (hunk 5 above)
Comment (dev-mlb) [reply src/test/java/emissary/server/api/HealthCheckActionTest.java:9]: Ticket created

File: src/test/java/emissary/server/api/MetricsActionTest.java (hunk 6 above)
Comment (dev-mlb) [reply src/test/java/emissary/server/api/MetricsActionTest.java:9]: Ticket created

File: src/test/java/emissary/server/api/PoolTest.java (hunk 7 above)
Comment (dev-mlb) [reply src/test/java/emissary/server/api/PoolTest.java:9]: Ticket created

File: src/test/java/emissary/server/api/VersionTest.java (hunk 8 above)
Comment (dev-mlb) [reply src/test/java/emissary/server/api/VersionTest.java:9]: Ticket created

File: src/test/java/emissary/server/mvc/DumpDirectoryActionTest.java (hunk 9 above)
Comment (dev-mlb) [reply src/test/java/emissary/server/mvc/DumpDirectoryActionTest.java:6]: Ticket created

File: src/test/java/emissary/server/mvc/EnvironmentActionTest.java (hunk 10 above)
Comment (dev-mlb) [reply src/test/java/emissary/server/mvc/EnvironmentActionTest.java:6]: Ticket created

File: src/test/java/emissary/server/mvc/NamespaceActionTest.java (hunk 11 above)
Comment (dev-mlb) [reply src/test/java/emissary/server/mvc/NamespaceActionTest.java:6]: Ticket created

File: src/test/java/emissary/server/mvc/ShutdownActionTest.java (hunk 12 above)
Comment (dev-mlb) [reply src/test/java/emissary/server/mvc/ShutdownActionTest.java:6]: Ticket created

File: src/test/java/emissary/server/mvc/ThreadDumpActionTest.java (hunk 13 above)
Comment (dev-mlb) [reply src/test/java/emissary/server/mvc/ThreadDumpActionTest.java:6]: Ticket created

File: src/test/java/emissary/server/mvc/TransferDirectoryActionTest.java (hunk 14 above)
Comment (dev-mlb) [reply src/test/java/emissary/server/mvc/TransferDirectoryActionTest.java:6]: Ticket created

**Pull request merged**